samples/gke_enable_logging.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
 spec:
   severity: high
   match:
-    target: # {"$ref":"#/definitions/io.k8s.cli.setters.target"}
+    ancestries:
```

Comment:
nit: is there a reason some ancestries/targets are defined as indented values vs. others that are abbreviated flow collections like `["org/**"]`

Reply:
not that I'm aware of; it predates me. I believe it's the same from the yaml perspective?